lib/services/ngsi/entities-NGSI-LD.js

```diff
@@ -55,7 +55,7 @@ const NGSI_LD_URN = 'urn:ngsi-ld:';
  *                                   format
```

Comment:
CHANGES_NEXT_RELEASE entry about the changes in this PR should be included

Reply:
Fixed f5f84b0